### PR TITLE
#1180 Use Github links when tweeting contributions

### PR DESCRIPTION
**Pull request by carlosmiranda**
#1180: Github link instead of Twitter handle is used when project contributions are tweeted out (Zerocracy does not even keep track of Twitter names). Added test to verify the message.

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#1516](https://github.com/zerocracy/farm/issues/1516) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#1516](https://github.com/zerocracy/farm/issues/1516) is assigned to @krzyk[/z](https://www.0crat.com/u/krzyk), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/2130345013); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: src/test/resources/com/zerocracy/bundles/tweets_contribution/_after.groovy [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+import com.jcabi.xml.XML
+import com.zerocracy.Project
+import com.zerocracy.entry.ExtTwitter
+import org.hamcrest.MatcherAssert
+import org.hamcrest.Matchers
+
+def exec(Project project, XML xml) {
+  MatcherAssert.assertThat(
+    ((ExtTwitter.Retrievable) new ExtTwitter(farm).value()).tweets(),
```
Comment (krzyk) [inline src/test/resources/com/zerocracy/bundles/tweets_contribution/_after.groovy]: @carlosmiranda you don't have `farm` defined anywhere

**Comment by carlosmiranda** [comment]
@krzyk fixed, please check

File: src/test/resources/com/zerocracy/bundles/tweets_contribution/_after.groovy [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+import com.zerocracy.entry.ExtTwitter
+import org.hamcrest.MatcherAssert
+import org.hamcrest.Matchers
+
+def exec(Project project, XML xml) {
+  MatcherAssert.assertThat(
+    ((ExtTwitter.Retrievable) new ExtTwitter(binding.variables.farm)
+        .value()).tweets(),
+    Matchers.contains(
+     'a contribution of $1 from http://github.com/carlosmiranda'
```
Comment (krzyk) [inline src/test/resources/com/zerocracy/bundles/tweets_contribution/_after.groovy]: @carlosmiranda the message is actually `The project TWEETSCON received a monetary contribution of $1.00 from https://github.com/carlosmiranda; many thanks for your support!`, so here you would need to use another matcher inside the `contains`

**Comment by carlosmiranda** [comment]
@krzyk pushed new commit

**Comment by carlosmiranda** [comment]
@krzyk weird that it's passing Travis and locally. Appveyor is correct...

**Comment by krzyk** [comment]
@rultor merge

**Comment by g4s8** [comment]
@rultor merge

**Comment by carlosmiranda** [comment]
@g4s8 can we try again? I can't see how this is related, passed on CI too.

**Comment by carlosmiranda** [comment]
@g4s8 looks like this is breaking some other recent changes, let me check

**Comment by carlosmiranda** [comment]
@g4s8 let's try again, looks like it's working again after merging the latest code.

**Comment by g4s8** [comment]
@rultor merge

**Comment by krzyk** [comment]
@0crat out

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @krzyk[/z](https://www.0crat.com/u/krzyk), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
> @0crat out ([here](https://github.com/zerocracy/farm/issues/1516#issuecomment-410476715))

@krzyk The job [#1516](https://github.com/zerocracy/farm/issues/1516) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by ypshenychka** [comment]
@krzyk According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
>The code reviewer found at least three problems in the code.
Comments were mostly about design problems, not cosmetic issues.

Only two issues were found during code review. 
Please confirm that you'll try to find at least three major problems while future reviews.

**Comment by krzyk** [comment]
@ypshenychka I confirm

**Comment by 0crat** [comment]
Order was finished, quality is "acceptable": +15 point(s) just awarded to @krzyk[/z](https://www.0crat.com/u/krzyk)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Pull request merged**